aries_cloudagent/commands/upgrade.py

```diff
@@ -394,6 +404,61 @@ async def update_existing_records(profile: Profile):
     pass
 
 
+async def update_routing_keys(profile: Profile):
+    """Update routing keys previously stored in wallet.
+
+    This update step will transform the stored routing keys into did:key values
+    from raw base58 encoded values.
+
+    Steps:
+        for each mediation record stored in the wallet:
+            ensure the stored routing_keys list is formated as did:keys
+            save the record if modified
+        for each routing record stored in the wallet:
+            ensure the stored recipient keys are formated as did:keys
+            save the record if modified
+    """
```

Comment:
This is the routine we've added for updating the routing keys. The goal of doing the upgrade this way was to ensure that we don't have to keep base58 -> did:key translation code on the `MediationRecord` and `RouteRecord` hydration methods from now until the rest of forever. Is this the right approach? Do we anticipate the upgrade script will continue to be used in this way in the future? Does this logic belong here or should it go somewhere else?

cc @shaangill025 @swcurran